src/utilities/helperFunctions.js

```diff
@@ -27,7 +27,17 @@ const genActionButton = (action_id, text, style = null) => {
   return btn;
 };
 
+/**
+ * Extracts a channel ID or user ID from a Slack-provided blocks array.
```

Comment:
The function should probably either only accept a single block, or the comment here should probably read `Extracts a channel ID or user ID from the first element of a Slack-provided blocks array.`.

The way it currently reads would imply it would pull the ID from any element of the blocks array.

Reply:
Accepting only a single block would require more code duplication at all 3 uses of this function. Therefore, that change would not make sense. Instead, I changed the comment to your suggested wording.